<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTranspositionGroup</class>
 <widget class="QDialog" name="DialogTranspositionGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Диапазон генерации количества элементов в перестановке:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinNmin"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>До:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>От:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="spinNmax"/>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QCheckBox" name="btnMod2">
       <property name="text">
        <string>Вычислить четность</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="typeWrite">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="btnCount">
       <property name="text">
        <string>Вычислить количество беспорядков</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="btnMultiply">
       <property name="text">
        <string>Произведение подстановок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinWrite"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="btnCycle">
       <property name="text">
        <string>Вычислить цикловой тип</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="typeMultiply">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="typeInverse">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinMultiply"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="btnWrite">
       <property name="text">
        <string>Записать подстановку указанным способом</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinInverse"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="btnInverse">
       <property name="text">
        <string>Найти обратную</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Типы возможных заданий:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="btnOrder">
       <property name="text">
        <string>Вычислить порядок подстановки</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="typeCycle">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="typeMod2">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="typeCount">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="typeOrder">
       <property name="text">
        <string>Выбрать вид...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="spinOrder"/>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="spinCount"/>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinMod2"/>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinCycle"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Вид исходной подстановки</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
